todoapp/src/components/Header.vue

```diff
@@ -15,7 +18,7 @@ function handleAdd() {
   <div class="header">
     <h1 class="header-title">To do list</h1>
     <button @click="handleAdd" class="add-todo-btn">
-      <img :src="plus" />
+      <img :src="isMobile ? addTodo : addTodoDesktop" />
```

Comment:
This should act like a real button so please think about active state, disabled state and hover effect. Also you don't need to export the image from the design you can use an icon library for the "+" icon like [heroicons](https://heroicons.com/) and the green circle you will achieve using css